source/_integrations/fronius.markdown

```diff
@@ -13,7 +13,7 @@ ha_platforms:
   - sensor
 ---
 
-The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors in Home Assistant.
+The `fronius` sensor polls a [Fronius](https://www.fronius.com/) solar inverter, battery system or smart meter and presents the values as sensors in Home Assistant. Data is gathered directly through the system's datamanager card connected to your local network, this integration doesn't access the cloud. You will need to either set a static IP or assign a static DHCP lease, or alternatively access it through the local DNS name if your network is properly configured.
```

Comment:
This is now an introduction and an instruction in one. Please split the prerequisites from the introduction.

Reply:
Done, thanks for the observation.